packages/beacon-state-transition/src/merge/block/processExecutionPayload.ts

```diff
@@ -0,0 +1,108 @@
+import {GAS_LIMIT_DENOMINATOR, MIN_GAS_LIMIT} from "@chainsafe/lodestar-params";
+import {merge, ssz} from "@chainsafe/lodestar-types";
+import {byteArrayEquals, List} from "@chainsafe/ssz";
+import {CachedBeaconState} from "../../allForks";
+import {getCurrentEpoch, getRandaoMix} from "../../util";
+import {ExecutionEngine} from "../executionEngine";
+
+export function isExecutionEnabled(state: merge.BeaconState, body: merge.BeaconBlockBody): boolean {
+  return isMergeBlock(state, body) || isMergeComplete(state);
+}
+
+function isMergeBlock(state: merge.BeaconState, body: merge.BeaconBlockBody): boolean {
+  return (
+    !isMergeComplete(state) &&
```

Comment:
Something seems wrong here

should this be `||`?

Reply:
Python code is
```
def is_merge_block(state: BeaconState, body: BeaconBlockBody) -> bool:
    return not is_merge_complete(state) and body.execution_payload != ExecutionPayload()
```   

https://github.com/ethereum/consensus-specs/blob/0eb0a934a3/specs/merge/beacon-chain.md#is_merge_block